src/main/java/com/spikes2212/util/smartmotorcontrollers/SparkWrapper.java

```diff
@@ -8,16 +8,51 @@
 import com.spikes2212.control.TrapezoidProfileSettings;
 import com.spikes2212.util.UnifiedControlMode;
 
+/**
+ * A {@link SmartMotorController} which handles the communication between a {@link SparkBase} and the code.
```

Comment:
????
I would write something like 'A SmartMotorController representation of a Spark motor controller'